assets/src/edit-story/panels/elements/inputGroup.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Label from './label';
+import Group from './group';
+
+const Input = styled.input`
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	border: 1px solid;
+	border-radius: 4px;
+	font-size: 11px;
+	line-height: 16px;
+	width: 100px;
+`;
+
+function InputGroup( { type, label, value, isMultiple, onChange, postfix, disabled, min, max } ) {
```

Comment:
Why can't we make this more generic and we reuse it. Could this be moved to `compontents`?

Reply:
See https://github.com/ampproject/amp-wp/issues/4149

Reply:
I think this is best done by whoever implements the correct designs. The components might be split into many or joined into fewer or completely reorganised when implementing the designs, so I see no reason for early optimisations.